base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -275,44 +275,46 @@ where
                 },
                 // Incoming messages from the Comms layer
                 msg = transaction_stream.select_next_some() => {
-                    trace!(target: LOG_TARGET, "Handling Transaction Message");
-                    let (origin_public_key, inner_msg) = msg.into_origin_and_inner();
-                    let result  = self.accept_transaction(origin_public_key, inner_msg).await;
+                    let (origin_public_key, inner_msg) = msg.clone().into_origin_and_inner();
+                    trace!(target: LOG_TARGET, "Handling Transaction Message, Trace: Tag#{}", msg.dht_header.message_trace);
```

Comment:
call this message tag

Reply:
Apply this comment to your other logs to please

Reply:
As discussed, standardizing on the display implementation for MessageTag.